src/app/components/algorithms/implementation-view/implementation-view.component.ts

```diff
@@ -74,7 +74,7 @@ export class ImplementationViewComponent implements OnInit {
 
   onDeleteQuantumResource($event: DeleteParams): void {}
 
-  onDatalistConfigChanged(params: QueryParams): void {
+  onDatalistConfigChanged(params: any): void {
```

Comment:
Why are you not using a proper type?

Reply:
we adjusted that because the params changed to a Array which was not supported by the generic datalist which sent a comma separated string. We will adjust that so we can use the QueryParams again.